tests/test_arithmetics.rs

```diff
@@ -423,3 +424,227 @@ fn rich_comparisons_python_3_type_error() {
     py_expect_exception!(py, c2, "c2 >= 1", PyTypeError);
     py_expect_exception!(py, c2, "1 >= c2", PyTypeError);
 }
+
```

Comment:
Maybe it's better to have a comment on this mod?
E.g., `// Checks that undefined binary operations return NotImplemented.`

Reply:
Done.